### PR TITLE
Fix analysis warnings

### PR DESCRIPTION
**Pull request by brianpepin**

### CODE REVIEW

File: Source/Task/XTaskQueuePriv.h [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 #pragma once
```
Comment (jasonsandlin) [inline Source/Task/XTaskQueuePriv.h]: revert this one.  Matt made this change for xplat work

File: Source/Task/XTaskQueuePriv.h [hunk 2]
```diff
@@ -4,7 +4,7 @@
...
 
-#include <httpClient/async.h>
+#include "XTaskQueue.h"
```
Comment (jasonsandlin) [inline Source/Task/XTaskQueuePriv.h]: revert this one. Matt made this change for xplat work

Comment (brianpepin) [reply Source/Task/XTaskQueuePriv.h]: This header is shared with Windows though -- there is no httpClient there.  Was this one done wrong?

Comment (lubeltra) [reply Source/Task/XTaskQueuePriv.h]: Brian's change is correct

**Pull request merged**